tests/unit/Codeception/Lib/Driver/MysqlTest.php

```diff
@@ -51,6 +51,21 @@ public function tearDown()
         }
     }
 
+    /**
+     * If the method exists (PHPUnit 5) forward the call to the parent class, otherwise
+     * call `expectException` instead (PHPUnit 6)
+     */
+    public function setExpectedException($exception, $message = '', $code = null)
```

Comment:
There are some test classes not inheriting from `Codeception\Test\Unit`. That's why I didn't think of moving that method up. I think the trait is better. Unless you think that all those test should extend `Codeception\Test\Unit`.